### PR TITLE
Refactor UserStatsImporter

### PR DESCRIPTION
**Pull request by awead**
From our Friday refactoring session, we extracted the various tally
methods into a new class and removed the inheritance from
Sufia::UserStatImporter.

### CODE REVIEW

File: app/services/user_stats_importer.rb [hunk 1]
```diff
@@ -118,4 +79,17 @@ def create_or_update_object_stat(event, user_db_id, object, stat_class)
                      end
       updated_stat.save
     end
+
+    def create_or_update_user_stats(stats, user)
```
Comment (carolyncole) [inline app/services/user_stats_importer.rb]: Should we add a comment that this code was copied from Sufia::UserStat?

**Review by carolyncole — CHANGES_REQUESTED** [review]
@awead At some point we will need to add tests for the new class.  Did you want to add those now or should we add a ticket?  It should be a matter of taking some of the test from UserStats and putting them against UserStatisticReport.

**Comment by carolyncole** [comment]
@awead I'm not certain if this is an accurate code coverage drop.  Are you happy with the coverage on this PR?

**Comment by awead** [comment]
@cam156 yeah, I wasn't either. I'm happy if you are!

**Comment by carolyncole** [comment]
@awead if I am reading code coverage right it looks like import is not being tested
https://coveralls.io/builds/13847428/source?filename=app%2Fservices%2Fuser_stats_importer.rb#L12

**Pull request merged**